corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -107,6 +107,11 @@ function LocationModel(data, root, depth, func, withAllOption) {
   this.func = typeof func !== 'undefined' ? func : LocationModel;
   this.withAllOption = typeof withAllOption !== 'undefined' ? withAllOption : true;
 
+  this.enable = function() {
+    return _.every(this.children(), function(child) {
+      return child.name() === '_all' || child.can_edit()
```

Comment:
Expected indentation of 8 space characters but found 6. (indent)
Missing semicolon. (semi)